### PR TITLE
tested function

### PR DESCRIPTION
**Pull request by annasisoian**

### CODE REVIEW

File: src/getCoinCombination.test.js [hunk 1]
```diff
@@ -9,12 +9,46 @@ describe(`Function 'getCoinCombination':`, () => {
...
+  it(`should return [1, 1, 0, 2] if amount = 56`, () => {
+    expect(getCoinCombination(56)).toEqual([1, 1, 0, 2]);
+  });
+
+  it(`should return [2, 0, 1, 3] if amount = 87`, () => {
+    expect(getCoinCombination(87)).toEqual([2, 0, 1, 3]);
+  });
+
+  it(`should return [3, 0, 2, 7] if amount = 198`, () => {
+    expect(getCoinCombination(198)).toEqual([3, 0, 2, 7]);
```
Comment (coyotious) [inline src/getCoinCombination.test.js:48]: No need for this test. It's like the test above ([x, 0, x, x]).

File: src/getCoinCombination.test.js [hunk 2]
```diff
@@ -9,12 +9,46 @@ describe(`Function 'getCoinCombination':`, () => {
...
-  it(`should return [0, 0, 0, 0] if amount = 0`, () => {
+  it(`should return [3, 0, 2, 4] if amount = 123`, () => {
+    expect(getCoinCombination(123)).toEqual([3, 0, 2, 4]);
+  });
+
+  it(`should return [1, 1, 1, 1] if amount = 41`, () => {
+    expect(getCoinCombination(41)).toEqual([1, 1, 1, 1]);
+  });
+
+  it(`should return [2, 0, 2, 15] if amount = 397`, () => {
```
Comment (coyotious) [inline src/getCoinCombination.test.js:31]: No need for this test. It's like the test above ([x, 0, x, x]).

File: src/getCoinCombination.test.js [hunk 3]
```diff
@@ -9,12 +9,46 @@ describe(`Function 'getCoinCombination':`, () => {
...
+  it(`should return [2, 0, 0, 0] if amount = 2`, () => {
+    expect(getCoinCombination(2)).toEqual([2, 0, 0, 0]);
   });
 
-  // write more tests here
+  it(`should return [1, 1, 0, 2] if amount = 56`, () => {
+    expect(getCoinCombination(56)).toEqual([1, 1, 0, 2]);
+  });
+
+  it(`should return [2, 0, 1, 3] if amount = 87`, () => {
```
Comment (coyotious) [inline src/getCoinCombination.test.js:43]: No need for this test. It's like the test above ([x, 0, x, x]).

File: src/getCoinCombination.test.js [hunk 4]
```diff
@@ -9,12 +9,34 @@ describe(`Function 'getCoinCombination':`, () => {
...
+  });
 
+  it(`should return [1, 1, 0, 2] if amount = 56`, () => {
+    expect(getCoinCombination(56)).toEqual([1, 1, 0, 2]);
   });
 
-  // write more tests here
+  it(`should return [0, 0, 0, 0] if amount = 0`, () => {
+    expect(getCoinCombination(0)).toEqual([0, 0, 0, 0]);
+  });
```
Comment (Oleksandra-Pasenchenko) [inline src/getCoinCombination.test.js:41]: Try to think about some more boundary values and structure your tests (for example, from the smallest to the highest amount
